roles/containerd/defaults/main.yml

```diff
@@ -66,6 +69,10 @@ containerd_modprobe:
 
 # Entries for sysctl.
 containerd_sysctl:
-  - { state: "present", name: "net.bridge.bridge-nf-call-ip6tables", value: "1" }
+  - {
```

Comment:
This line is different than the others, maybe some autoformatting was the culprit

Reply:
cosmetic change, but yeap autoformat on save changed it. I've restored it as it was before.